helm/examples/azure/log-analytics/enable-loganalytics.sh

```diff
@@ -40,6 +40,23 @@ else
    echo "Tenant ID: ${tid}"
 fi
 
+echo "Confirming workspace resource group exists..."
+wsrgexist=$(az group exists -n $LOGANALYTICS_RESOURCE_GROUP)
+if [[ $? -ne 0 ]] || [[ $wsrgexist == false* ]]
```

Comment:
should this be "${wsrgexist}" =~ "^false*" or something similar?

Reply:
well you're the bash expert I rely on, but the current condition seems to work, what would be the advantage of the recommended notation? Is the gist of the recommendation to perform string-wise comparison and assert the string must start w/ the false substring?

Reply:
@Michael-Gardner not sure how I accidentally labeled this resolved. 
Anyway, the suggested change (and a few variations of it) did not work. Keeping as is for now.